deck-o-cards.js

```diff
@@ -1,33 +1,29 @@
-// 1. Make the function deck_o_cards assemble an array of cards using the provided suits and values arrays.
-// Each card in the deck should be an object formatted as: {suit: 'hearts', value: 'A'}
 function deck_o_cards() {
   var values = [2, 3, 4, 5, 6, 7, 8, 9, 10, 'Jack', 'Queen', 'King', 'Ace'];
   var suits = ['hearts', 'diamonds', 'clubs', 'spades'];
 
   var cards = []; // deck
   var shuffledCards = []; // deck shuffled
 
-  // Make 52 card objects and store them in the "cards" array
-  // Hint: use 2 for loops
-
-
-  // 2. Shuffle the cards
-  // Hint: shuffle function is already defined below
-
+  for (let i = 0; i < suits.length; i++) {
+    for (let x = 0; x < values.length; x++) {
+      cards.suits = suits[i];
+      cards.values = values[x];
```

Comment:
You were definitely headed down the right path, but you need to remember to create the card OBJECT with suit and value properties before pushing into the array. 
```js
let card = { suit: suits[i], value: values[x] }
```

Hopefully this morning's walkthrough helped clear some things up? If not - please let us know so we can answer your questions.